proto/spacemesh/v1/gateway_types.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+import "google/rpc/status.proto";
+
+message BroadcastPoetRequest {
+    bytes data = 1;  // signed binary transaction
```

Comment:
This one isn't a transaction, but an XDR-encoded data of `PoetProofMessage` defined in `github.com/spacemeshos/poet/service` Go package.